server/core/src/main/scala/sttp/tapir/server/interceptor/decodefailure/DecodeFailureHandler.scala

```diff
@@ -127,7 +127,7 @@ object DefaultDecodeFailureHandler {
       // we assume that the only decode failure that might happen during path segment decoding is an error
       // a non-standard path decoder might return Missing/Multiple/Mismatch, but that would be indistinguishable from
       // a path shape mismatch
-      case _: EndpointInput.PathCapture[_]
+      case _: EndpointInput.PathCapture[_] | _: EndpointInput.PathsCapture[_]
```

Comment:
@adamw I'm not sure why this happens only in the Play backend test, but this "fix" plus setting `badRequestOnPathErrorIfPathShapeMatches = true` fixed a failing test which was supposed to check for 4xx responses to a call with `/../` in the path.

Reply:
it's a new test? does it fail only for Play?

I think the tests should pass with the default options ... I would investigate why this fails with Play, but not with others - that's suspicious at least :)

Reply:
Yes, it's a new test - one that checks HEAD for resources: https://github.com/softwaremill/tapir/pull/2837/files#diff-9dd1d78e1946bf817bdd45376a7ac7502737617b90845fdab38fffbe3b4b6f39R604. Maybe Play's test interpreter is missing something.